packages/cli/src/commands/setup/deploy/templates/coherence.js

```diff
@@ -0,0 +1,54 @@
+import path from 'path'
+
+import { getPaths } from '../../../../lib'
+
+export const PROJECT_NAME = path.basename(getPaths().base)
+
+export const COHERENCE_YAML = (database) => {
+  return `
+api:
+  type: backend
+  url_path: "/api"
+  prod:
+    command: ["yarn", "rw", "serve", "api"]
+  dev:
+    command: ["yarn", "rw", "dev", "api"]
+  local_packages: ["node_modules"]
+
+  system:
+    cpu: 2
+    memory: 2G
+    health_check: "/graphql/health"
+
+  ${database}
+
+web:
+  type: frontend
+  assets_path: "web/dist"
```

Comment:
dockerfile required?

Reply:
for now we are just going to reply on nixpacks to build the images inside Coherence...